relay-server/src/services/project_cache.rs

```diff
@@ -556,16 +556,26 @@ struct ProjectCacheBroker {
     metric_outcomes: MetricOutcomes,
     // Need hashbrown because extract_if is not stable in std yet.
     projects: hashbrown::HashMap<ProjectKey, Project>,
+    /// Utility for disposing of garbage project data safely.
```

Comment:
It is safe, the purpose is not safety but performance (Drop handlers are expensive).

```suggestion
    /// Utility for disposing of expired project data efficiently.
```